centaur/test_cromwell.sh

```diff
@@ -109,7 +109,7 @@ cd "${RUN_DIR}"
 TEST_STATUS="failed"
 
 ENABLE_COVERAGE=${CENTAUR_SBT_COVERAGE} sbt centaur/it:compile
-CP=$(ENABLE_COVERAGE=${CENTAUR_SBT_COVERAGE} sbt "export centaur/it:dependencyClasspath" --error)
+CP=$(ENABLE_COVERAGE=${CENTAUR_SBT_COVERAGE} sbt "early(error)" "export centaur/it:dependencyClasspath")
```

Comment:
the syntax 😦

Reply:
https://github.com/sbt/sbt/issues/4088

Reply:
For some reason "-error" works, using that to ease the pain.  I think they are undecided on their UNIX-ness vs Java-ness